src/gpl/src/graphics.cpp

```diff
@@ -476,6 +476,26 @@ bool Graphics::populateMap()
   return true;
 }
 
+void Graphics::populateXYGrid()
+{
+  BinGrid& grid = nbVec_[0]->getBinGrid();
+  std::vector<Bin>& bin = grid.bins();
+  int x_grid = grid.binCntX();
+  int y_grid = grid.binCntY();
+
+  std::vector<int> x_grid_set, y_grid_set;
+  x_grid_set.push_back(bin[0].lx());
+  y_grid_set.push_back(bin[0].ly());
+
+  for (std::vector<Bin>::size_type x = 0; x < x_grid; x++) {
```

Comment:
Why are you using `std::vector<Bin>::size_type` when your `x_grid` limit for the loop is an `int`?